examples/apps/echo/README.md

```diff
@@ -0,0 +1,49 @@
+# Echo Demo Application
+
+This simple demo application echos back the same message sent from
+the client to the worker.
+
+## Using the Echo Command Line Client
+
+The command line client, `echo_client.py` sends a message to the worker,
+which echos back the same message back to the client.
+For command line options, type `./echo_client.py -h` from the
+`client` directory.
+
+To use:
+
+1.  If needed, update the Ethereum account and direct registry contract
+    information in `docker/Dockerfile.tcf-dev` and
+    `examples/common/python/connectors/tcf_connector.toml`
+2.  Follow instructions in the "Docker-based Build and Execution" section of
+    the [build document](../../../BUILD.md#dockerbuild) through step 4
+    (activating a virtual environment)
+3.  Terminal 1 is running the TCF Enclave Manager and Listener with
+    `docker-compose` . Terminal 2 is running the Docker container shell
+    with the `(_dev)` prompt
+4.  In Terminal 2 run `cd $TCF_HOME/examples/apps/echo/client`
+5.  In Terminal 2 install the Solidity compiler as follows:
```

Comment:
Solidity compiler is already installed in dockerfile in docker/Dockerfile.tcf-dev

Reply:
No it is not. It is not there for the bash shell and it has to be installed manually.
For example:

```
$ sudo docker exec -it tcf bash
root@f48d605bea9e:/project/TrustedComputeFramework/tools/build#  source _dev/bin/activate
(_dev) root@f48d605bea9e:/project/TrustedComputeFramework/tools/build# ls -a ~/.py-solc
ls: cannot access '/root/.py-solc': No such file or directory
(_dev) root@f48d605bea9e:/project/TrustedComputeFramework/tools/build# echo $SOLC_BINARY
```

Reply:
So although solc is installed from dockerfile file, it's not accessible from other shell. May be we coudl eitehr insatll solc in /usr/bin or add the solc path of /etc/environment  or ~/.bashrc so that it's accessible from all bash shell. @Ram-srini Can this be done ?

Reply:
@danintel for docker based build we don't need to install solc compiler again.
(_dev) root@0840863d8f74:/project/TrustedComputeFramework/tools/build# pip list web3 |grep web3
web3                5.0.2
(_dev) root@0840863d8f74:/project/TrustedComputeFramework/tools/build# echo $SOLC_BINARY
/root/.py-solc/solc-v0.4.25/bin/solc
(_dev) root@0840863d8f74:/project/TrustedComputeFramework/tools/build# ls $SOLC_BINARY
/root/.py-solc/solc-v0.4.25/bin/solc

Reply:
I experimented around on different machines.  I found this stuff works only on a machine with direct Internet access. It does not work behind the Intel corporate proxy.  I created this JIRA bug for this problem:

Reply:
I found the problem. The `BUILD.md` instructions are missing:
`docker-compose up --build`
Otherwise the `docker/Dockerfile.tcf-dev` file is not ran.
This occurs if an old Docker image still exists.
I am fixing steps in `docker/Dockerfile.tcf-dev` as the OpenSSL build fails.  I think the problem is `$THREADS` is causing a failure.